### ISSUE TITLE
Review: Release 3.2

### ISSUE THREAD

**Issue by 2amarkina** [blockframes/blockframes#5938]
## Environment
- OS: MacOS Big Sur
- Browser (& version if possible): Safari 14.0.3 (16610.4.3.1.7)

### Tested on Archipel Content
- Url: https://staging.archipelcontent.com

### Cross-App **Schedule Demo section** 
Now we've got this screen after sending a request. 
![Снимок экрана 2021-06-02 в 14.03.57.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/4ae91b51-026f-4794-bcc5-8f2f112aca63)

- [x] We can go for using different states for this button: 

1. Send Request ![Снимок экрана 2021-06-02 в 14.06.07.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/8e9ddd41-2276-49dc-a9c5-08c9d6f5cf34)
2. Sending Request... ![Снимок экрана 2021-06-02 в 14.06.07.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/676cd1bb-1796-45bc-8fd6-112ac35ec0fd)
3. Request Sent ![Снимок экрана 2021-06-02 в 14.11.03.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/102fdb74-c50b-4044-acf4-77954ee411ef)

### Cross-App **Forgot Password**

- [x] Page is not adapted to the Log In / Sign Up style. Maybe not for this Release, but should be seen with @stephanie-cascade8  

![Снимок экрана 2021-06-02 в 14.14.14.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/9ee91de8-f742-40a3-8f64-7bab1dd532bf) 

### Cross-App **Create your account**

- [x] Need to have a possibility to Log In on this page. 

"Already have an account? Log in here" 
![Снимок экрана 2021-06-02 в 14.15.53.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/9e4b639d-8158-4065-be57-215685890840)

### Cross-App **Movie Tunnel**

- [ ] **_Business team check needed_**
For the following elements: _Director's Description, Cast Member's Description, Crew Member's description_ introduce the validator maxLength<200. Reason: keep this field rather small to avoid eventual scrolling. 

Summary Page
- [x] Add info about Production Status in Reference 
- [ ] Make Production Status mandatory 
![Снимок экрана 2021-06-02 в 14.39.34.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/35f66cca-2e2e-485a-a1af-66a4793b2647)
_For AC production status will always remain Released_

- [ ] Poster and Banner Images are missing, but they are uploaded
![poster:banner.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/3f24d26b-72fb-4f0f-9ca2-73e9e168355c)

- [x] Clicking on Poster Image should lead to the Film Poster section 
- [ ] Clicking on Banner Image should lead to Film Banner section 

- [x] Clicking on Missing information about director should lead to Director(s) section
![director.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/bb7b8104-7a01-473b-9df3-2afbc403a38a)

- [x] Clicking on Key Assets missing info makes the navigation buttons "fly". Happens only with Key Assets (all good for logline, synopsis)
![key assets.gif] (https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/66d36265-4bc9-46b1-ba73-f43c65030120)

- [ ] Missing Info on Selection and Reviews leads to the Artistic Team ! should lead to Festivals & Awards. Same problem for Other Festivals
![selections.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/ba4d3335-806f-4d5b-821a-3ee7bc8c127e)

- [ ] Same problem of misleading : Film Reviews missing leads to the bottom of Artistic Team page and should lead to the Film reviews section
![film review.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/81141dd5-c421-411b-be4e-d272dd372fd9)

- [ ] Clicking on Box office missing makes the navigation buttons "fly". Same for "Rating"
![box office.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/7037e4e5-e68c-47f0-994d-e08fe608eb62)

- [ ] Clicking on Qualification missing should lead to Qualification section

- [ ] Clicking on Target Audience Missing should lead to Positioning section. Same for Social Responsibility Goals Missing

- [ ] Files are not uploaded. On the Summary page Presentation Deck is Missing, event if document was uploaded. Same for Scenario and Moodboard
![files.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/dc765dce-cd60-44e6-8382-f6caef8fb6d0)

N.B. After Saving the images and files appeared on the summary page. But showing "missing" is confusing (as user successfully added files). Proposition, maybe use the same solution we had for the Still Photos? Means show only the name of section "Presentation Deck" for example. 
![Снимок экрана 2021-06-02 в 15.05.29.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/b9c244c9-da45-4651-9e45-0fc16adecfdb) 

- [x] Following the [Modals documentation](https://www.notion.so/cascade8/Modals-warnings-0c8d7ccd3bab4c1c904da5c4b8aa48e4) let's homogenize the modals' structure everywhere. 
1. add close button
2. put Main CTA "Save & Exit" above "Close without saving". 
![Снимок экрана 2021-06-02 в 15.11.05.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/8f1c6742-6907-45da-850a-916e651117c7)

### Cross-App **Film Page / Dashboard**

- [x] **Problems exists only on safari !** Background color problem: should go from Primary 200 to transparent. Should be exactly like on Chrome. If needed, check with @stephanie-cascade8 
What we have: ![Снимок экрана 2021-06-02 в 15.15.46.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/1d5a4204-c74c-4993-bc18-f5c066eee8c5)

What should have: ![Снимок экрана 2021-06-02 в 15.20.38.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/25a8a31b-1766-4e02-891d-0b2c4729f74d)

### Cross-App **Profile / Organization**
- [x] background color problem (Safari only). Should be exactly the same as on Chrome
**Safari view** 
![Снимок экрана 2021-06-03 в 10.18.56.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/a73b79bd-3fd4-435d-91d2-a392f6debe06)
**Chrome view**
![Снимок экрана 2021-06-03 в 10.21.50.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/32cee0df-0e97-4288-9c78-a4f8a021d17d)

### Cross-App **Wishlist**

- [ ] **business team confirmation needed** As the wishlist is shared within Company, there is a proposition to change the title by "Company's Wishlist" when there is something in Wishlist
![Снимок экрана 2021-06-02 в 16.57.30.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/e131bfe7-1681-44e2-a083-c8fbadb4d23d)

- [ ] **business team confirmation needed** As there is not enough space, change the title "My Wishlist" by simple "Wishlist" 
![Снимок экрана 2021-06-02 в 16.58.43.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/a1ff9255-063f-4866-b91b-ebceb1c6ece6)

**Comment by 2amarkina** [comment]
### Environment
OS: MacOS Big Sur
Browser (& version if possible): Safari 14.0.3 (16610.4.3.1.7)

### Tested on Archipel Content
Url: https://staging.archipelcontent.com

### **Avails Page** 

- [ ] As Terms and Rights are the part of "Vocabulary" in Capital Letter Documentation, change the text "Search for new Terms and Rights" 
![Снимок экрана 2021-06-02 в 17.15.17.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/12fbcafe-1538-4097-af5f-3df9cffd57f9) 


### **Avails Page MAP** 
- Map doesn't work. Filters are applied correctly, snackbar appears, but Rights Card is not created, the country is not selected neither... But at the same time, "select all countries" works well
![map.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/44b42cb3-a112-47c3-9e0b-459b008b54df)

![select all.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/784ffb5e-35c7-4aeb-8e08-4d9f75dd0bca)

### **Avails Page CALENDAR** 

- [ ]  If possible, not allow user to set "end" for the same month/year as "start". If not, we receive the card with "Exclusive Rights from 05/01/2026 until 05/01/2026". 

![start:end.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/0c0bf83a-a989-4dad-87e0-8beb59e0e2d4)

### **Selection Page**
- [ ] Edit button should not only lead to Film Page, but should have the same behavior as edit btn in Preselection Rights Card (fill in avails criteria, show selected countries). 

If this is not possible, two possible solutions: 
1(_preferable_). remove edit and remplace it by new icon with "show more" sense (ask @stephanie-cascade8 and @jdrns ) and make a user land on the needed Rights Card 
![edit selection.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/0019b3b9-b62b-49d0-b41f-fb862fb2b112)

2. remove edit at all

**Modals "Learn how it works" + "Too many countries"**

- [ ] The close icon is not placed the same. Need to homogenize. Privilege one with more space around (Learn how it works)


### **Dashboard Film Page**
- [ ] Empty States needed for Sales Revenue section (Total Gross Receipts, Total Sales)
![Снимок экрана 2021-06-02 в 17.27.59.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/594218d6-5404-4795-951f-26513caad790)

### **Dashboard My Titles**
- [ ] When activate/deactivate filters, the column with status appears becomes the first column, and should be the last 
![filters tables.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/093aa45d-f4f2-40c9-87d0-35a1ae5a26bd)

**Comment by 2amarkina** [comment]
## Your environment
OS: MacOS Big Sur
Browser : Chrome (91.0.4472.77)

## Review item
Connexion process

### Localization
- App name: Archipel Market
- Page: Connexion Page
- Url: https://staging.archipelmarket.com/c/organization/join-congratulations
- User used (if anonymized): 
dev+cassandre-mko@blockframes.io

### Context
- [ ] When tried to login, strange bug appeared. If email was not verified, need to put "waiting" icon. 
![Снимок экрана 2021-06-03 в 11.41.38.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/21d1d80a-0fe9-4899-b91d-ee66a2051805)

N.B. This bug doesn't always appear. Tried to connect 3 times, got it 2 out of 3 times.

## Your environment
OS: MacOS Big Sur
Browser : Safari 14.0.3 (16610.4.3.1.7)

## Review item
Screening Page leaving Modal

### Localization
- App name: Archipel Market
- Page: Screening Page
- Url: https://staging.archipelmarket.com/c/o/marketplace/event/zkjQ1kCJ09QNO0KW88Sn/session
- User used (if anonymized): 
dev+trevor-kcu@blockframes.io 

### Context

- Need to homogenize the modal windows everywhere following [Modals documentation.](https://www.notion.so/cascade8/Modals-warnings-0c8d7ccd3bab4c1c904da5c4b8aa48e4)
- Following the [Modals Text Inventory ](https://docs.google.com/spreadsheets/d/1N06uryQf5J7g56kyenJgpZ_HtiXmYb_Evf798E4fwsY/edit#gid=678850512) change the text

### Fix
- [ ] add close button 
- [ ] title "You are about to leave this screening."
- [ ] main text "You might not be able to come back as its access is time-limited."
- [ ] main CTA "Leave anyway" should be above secondary CTA "Stay"

### Images
This modal 
![Снимок экрана 2021-06-03 в 10.47.48.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/d9267a22-acc5-4281-8008-fec8f5247297)

Should have this structure 
![Снимок экрана 2021-06-03 в 10.48.14.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/24ffadc0-7f39-43e8-917a-d1e790aa76d8)


## Your environment
OS: MacOS Big Sur
Browser : Safari 14.0.3 (16610.4.3.1.7)

## Review item
Image Preview 

### Localization
- App name: Archipel Market
- Page: Event creation page, My Files, Image carousel etc.
- Url: 
https://staging.archipelmarket.com/c/o/dashboard/event/20vZ75BpRL4UcuJFokHG/edit
- User used (if anonymized): 
dev+trevor-kcu@blockframes.io 

### Context
Homogenize the behavior  of modal

### Fix
- [ ] add close button 

### Images
Image Modal example 
![Снимок экрана 2021-06-03 в 11.15.27.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/053ea9d1-d371-4c35-9fb7-e6936bd08ef8)

## Your environment
OS: MacOS Big Sur
Browser : Safari 14.0.3 (16610.4.3.1.7)

## Review item
Meeting Page leaving Modal

### Localization
- App name: Archipel Market
- Page: Meeting Page
- Url: https://staging.archipelmarket.com/c/o/marketplace/event/20vZ75BpRL4UcuJFokHG/session
- User used (if anonymized): 
dev+trevor-kcu@blockframes.io 

### Context

- Need to homogenize the modal windows everywhere following [Modals documentation.](https://www.notion.so/cascade8/Modals-warnings-0c8d7ccd3bab4c1c904da5c4b8aa48e4)
- Following the [Modals Text Inventory ](https://docs.google.com/spreadsheets/d/1N06uryQf5J7g56kyenJgpZ_HtiXmYb_Evf798E4fwsY/edit#gid=678850512) change the text

### Fix
- [ ] add close button 
- [ ] title "You are about to leave this meeting.."
- [ ] main text "You might not be able to come back as its access is time-limited."
- [ ] main CTA "Leave anyway" should be above secondary CTA "Stay"

### Images
This modal 
![Снимок экрана 2021-06-03 в 11.49.50.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/ca884942-4c5c-45f0-8a0a-568311db67f0)

Should have this structure 
![Снимок экрана 2021-06-03 в 10.48.14.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/24ffadc0-7f39-43e8-917a-d1e790aa76d8)

## Your environment
OS: MacOS Big Sur
Browser : Chrome (91.0.4472.77)

## Review item
Meeting Page (Buyer View)

### Localization
- App name: Archipel Market
- Page: Meeting Page (Buyer View)
- Url: https://staging.archipelmarket.com/c/o/marketplace/event/20vZ75BpRL4UcuJFokHG/session
- User used (if anonymized): 
dev+cassandre-mko@blockframes.io

### Context

- [ ] The horizontal scrollbar should not appear

### Images
![horizontal scroll.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/1f3e3af7-fc28-4216-af05-3d377162f76e)


## Your environment
OS: MacOS Big Sur
Browser : Safari 14.0.3 (16610.4.3.1.7)

## Review item
Meeting Page, parameters Modal

### Localization
- App name: Archipel Market
- Page: Meeting Page
- Url: https://staging.archipelmarket.com/c/o/marketplace/event/20vZ75BpRL4UcuJFokHG/session
- User used (if anonymized): 
dev+trevor-kcu@blockframes.io 

### Context

- Need to homogenize the modal windows everywhere following [Modals documentation.](https://www.notion.so/cascade8/Modals-warnings-0c8d7ccd3bab4c1c904da5c4b8aa48e4)

### Fix
- [ ] add close button 
- [ ] main CTA "Unblock autoplay" should be above secondary CTA "Cancel"

This modal 
![Снимок экрана 2021-06-03 в 12.07.49.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/00736ffe-8630-4a5a-9653-60f0b706ea26)

Should have this structure 
![Снимок экрана 2021-06-03 в 10.48.14.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/24ffadc0-7f39-43e8-917a-d1e790aa76d8)

## Your environment
OS: MacOS Big Sur
Browser : Safari 14.0.3 (16610.4.3.1.7)

## Review item
Full Screen btn

### Localization
- App name: Archipel Market
- Page: Meeting Page for shared elements, My Files ...
- Url: https://staging.archipelmarket.com/c/o/dashboard/files
- User used (if anonymized): 
dev+trevor-kcu@blockframes.io 

### Context

- [ ]  Putting a document / image in full size is not working
![full screen.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/67bb189f-abc3-47e5-a5e0-7e0849e3aa4d)

![seller meeting.gif](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/7813a7ab-3c40-4c2f-b1e5-6a75c30bd424)

**Comment by RemcoSimonides** [comment]
@Jackseed 
> - [ ] **business team confirmation needed** As the wishlist is shared within Company, there is a proposition to change the title by "Company's Wishlist" when there is something in Wishlist
![Снимок экрана 2021-06-02 в 16.57.30.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/e131bfe7-1681-44e2-a083-c8fbadb4d23d)

> - [ ] **business team confirmation needed** As there is not enough space, change the title "My Wishlist" by simple "Wishlist" 
![Снимок экрана 2021-06-02 в 16.58.43.png](https://images.zenhubusercontent.com/5dd54528c0ccd300013672c3/a1ff9255-063f-4866-b91b-ebceb1c6ece6)

**Comment by RemcoSimonides** [comment]
@Jackseed 
> ### Cross-App Movie Tunnel
 Business team check needed
For the following elements: Director's Description, Cast Member's Description, Crew Member's description introduce the validator maxLength<200. Reason: keep this field rather small to avoid eventual scrolling.

**Comment by RemcoSimonides** [comment]
> ## Your environment
> OS: MacOS Big Sur
> Browser : Chrome (91.0.4472.77)
> 
> ## Review item
> Connexion process
> 
> ### Localization
> * App name: Archipel Market
> * Page: Connexion Page
> * Url: https://staging.archipelmarket.com/c/organization/join-congratulations
> * User used (if anonymized):
>   [dev+cassandre-mko@blockframes.io](mailto:dev+cassandre-mko@blockframes.io)
> 
> ### Context
> * [x]  When tried to login, strange bug appeared. If email was not verified, need to put "waiting" icon.
>   ![Снимок экрана 2021-06-03 в 11.41.38.png](https://camo.githubusercontent.com/8284eecd52a30bf8731a5bf00d0740cfee8451f6f833251cf0552efbbc3073df/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3564643534353238633063636433303030313336373263332f32316431643830612d306665392d343839392d623931642d656536366132303531383035)
> 
> N.B. This bug doesn't always appear. Tried to connect 3 times, got it 2 out of 3 times.

Should not be there anymore. Let's see if it's gone after next deploy to staging (new deploy will happen today)

**Comment by Jackseed** [comment]
@RemcoSimonides yes let's go for this one, empty state text to be changed as well for “Your Company’s Wishlist is empty.” instead of "Your Wishlist is empty."

> @Jackseed
> 
> > * [ ]  **business team confirmation needed** As the wishlist is shared within Company, there is a proposition to change the title by "Company's Wishlist" when there is something in Wishlist
> >   ![Снимок экрана 2021-06-02 в 16.57.30.png](https://camo.githubusercontent.com/37efcb2f02ce502700d1c636621143ff0fca3c8226c68118a837ce1fc4162f7e/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3564643534353238633063636433303030313336373263332f65313331626665372d313638312d343465322d613038332d633866626164623464323364)
> 
> > * [ ]  **business team confirmation needed** As there is not enough space, change the title "My Wishlist" by simple "Wishlist"
> >   ![Снимок экрана 2021-06-02 в 16.58.43.png](https://camo.githubusercontent.com/2f3b481ce77c7c7c212c4d5f94030233c236208b7d97bb243ffcb9464e15e033/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3564643534353238633063636433303030313336373263332f61316666393235352d303633662d343836362d623931622d656263656231633665636536)

**Comment by 2amarkina** [comment]
> > ### Cross-App **Movie Tunnel**
> > 
> > * [ ]   **_Business team check needed_**
> >   For the following elements: _Director's Description, Cast Member's Description, Crew Member's description_ introduce the validator maxLength<200. Reason: keep this field rather small to avoid eventual scrolling.
> 
> @2amarkina I'm not sure with this proposition as the current behaviour works well with too long text: it adds "..." & user can read the whole text within the modale when clicked.
> See this movie for example: https://staging.archipelmarket.com/c/o/marketplace/title/bR4fTHmDDuOSPrNaz39J/main
> The current description is up to 376 characters and I don't see any issue.
> ![image](https://user-images.githubusercontent.com/14964443/120650402-0bedf680-c47e-11eb-80ff-b69394e7acb3.png)
> ![image](https://user-images.githubusercontent.com/14964443/120650409-0ee8e700-c47e-11eb-8e0f-fcb4de8764ca.png)
> 
> Last thing, limiting to 200 characters could in fact be annoying for users, having fill in these kind of forms myself, you usually copy paste some existing documentation. If your description is let's say 500 characters, then user will just drop this input cause s/he can't copy/paste anymore.

Yes, ok for Marketplace view, it was mostly about the Movie Tunnel part. Ok for possible long description. The other solution I was thinking over was to make this field a little bit bigger / higher to prevent the scroll appear too early. Also, logline which is limited to 350 characters becomes bigger than Description that could go till 500 
<img width="1088" alt="Снимок экрана 2021-06-03 в 16 16 54" src="https://user-images.githubusercontent.com/55499036/120660048-398b6d80-c487-11eb-9e53-2c7bf295ffeb.png">
<img width="1090" alt="Снимок экрана 2021-06-03 в 16 08 43" src="https://user-images.githubusercontent.com/55499036/120660066-3e502180-c487-11eb-8329-30190e457ca5.png">

**Comment by 2amarkina** [comment]
> Hum then what about switching the 2 components: small one for logline and bigger one for descriptions?

I like this solution, but let's make the logline a bit smaller in such case. As the Description is still not filled in so often, we do not need to made this page too long neither. Would propose something between 100 and 120 px height, but need to check with @stephanie-cascade8 if it's ok.

**Comment by RemcoSimonides** [comment]
Hee @2amarkina , if you have time today; can you check if the issues you mentioned are fixed (and resolved correctly)? If yes, you can tick the box. if not, you can leave the checkbox unchecked open. Let me know when you're finished so I can see those issues.

**Comment by 2amarkina** [comment]
**Retested with Safari (14.0.3 (16610.4.3.1.7)**

- [ ] put the production status in capital "Released". Same for other possible statuses: In Development, In Production, In Post-Production, Completed, Released. 
<img width="347" alt="Снимок экрана 2021-06-08 в 11 14 29" src="https://user-images.githubusercontent.com/55499036/121158820-f21c2d00-c84a-11eb-9d49-22426e335e0d.png">

- [ ] If no production status, put "missing" with yellow color as for other fields. "Missing" should be clickable. A click should lead to the "First Step".
<img width="478" alt="Снимок экрана 2021-06-08 в 11 15 46" src="https://user-images.githubusercontent.com/55499036/121159172-40c9c700-c84b-11eb-9f2f-4ca15f080d13.png">

- [ ] leads to poster section, but buttons are still "flying" 
![btns](https://user-images.githubusercontent.com/55499036/121159920-ebda8080-c84b-11eb-941f-eebb686f6fba.gif)

- [ ] still doesn't lead to film banner, buttons "fly"
![fly](https://user-images.githubusercontent.com/55499036/121160417-5be90680-c84c-11eb-9121-3d76bbbe8c03.gif)

- [ ] key assets - ok, but buttons "fly" 

- [ ] **!!!** all "selection and review" part still leads to "artistic team". should lead to "selection and reviews". 
Festival & Award + Other Festival & Award : lead to the top of artistic team 
Film Review : leads to to the artistic team filmography 
![selections](https://user-images.githubusercontent.com/55499036/121161068-eb8eb500-c84c-11eb-85e6-14fbe5c7a4e0.gif)

- [ ] **!!!** "Search and Buy Rights" CTA doesn't work anymore. If other actions were made, like clicking on "Main" and then click on "Search and Buy Rights" starts working. 
![rights cta](https://user-images.githubusercontent.com/55499036/121162281-14fc1080-c84e-11eb-8671-c179d3a5d93e.gif)

Calendar doesn't work properly on Safari (ok on chrome). 
- [ ] create a snackbar when period is chosen
- [ ] create a preselected rights card when period is chosen
![calendar](https://user-images.githubusercontent.com/55499036/121163817-622cb200-c84f-11eb-8ed0-bac1fe699e24.gif)

- [ ] Really great for Edit btn on the selection page !!! 🥇  But just a little remark, could we make it appear that we could see also the title of a card ? (here is a capture) 
like that: 
<img width="1393" alt="Снимок экрана 2021-06-08 в 11 51 06" src="https://user-images.githubusercontent.com/55499036/121164390-d9fadc80-c84f-11eb-8c38-8b2e731153a4.png">

and now we've got that:
![edit](https://user-images.githubusercontent.com/55499036/121164624-fe56b900-c84f-11eb-8459-0169632cab4e.gif)

- [ ] new strange error with video player 
<img width="1393" alt="Снимок экрана 2021-06-08 в 11 52 38" src="https://user-images.githubusercontent.com/55499036/121164759-10d0f280-c850-11eb-8f87-386ed19cfdb6.png">

Put it here also: 
- modals still need to be changed (low priority),
- text for wishlist (medium priority, cross-app): 

- [ ] empty wishlist page "Your Company’s Wishlist is empty."
- [ ] wishlist with content "Company's Wishlist" 
- [ ] menu text "Wishlist" 

- full size for images, docs etc. doesn't work (high priority)
